src/morphology/english/irregularNouns.js

```diff
@@ -0,0 +1,296 @@
+	[ "metra", "metrae", "metræ" ],
+	[ "microfauna", "microfaunae", "microfaunæ" ],
+	[ "microflora", "microflorae", "microfloræ" ],
+	[ "mina", "minae", "minæ" ],
+	[ "minutia", "minutiae", "minutiæ" ],
+	[ "molecula", "moleculae", "moleculæ", "moleculas" ],
+	[ "monomania", "monomaniae", "monomaniæ" ],
+	[ "mora", "morae", "moræ" ],
+	[ "natura", "naturae", "naturæ" ],
+	[ "nebula", "nebulae", "nebulæ" ],
+	[ "onomatopoeia", "onomatopoeiae", "onomatopoeiæ" ],
+	[ "oötheca", "oöthecae", "oöthecæ" ],
+	[ "palaestra", "palaestrae", "palaestræ" ],
+	[ "panacea", "panaceae", "panaceæ", "panaceas" ],
+	[ "papilla", "papillae", "papillæ" ],
+	[ "papilloma", "papillomae", "papillomæ", "papillomas" ],
+	[ "parabola", "parabolae", "parabolæ", "parabolas" ],
+	[ "paraphilia", "paraphiliae", "paraphiliæ", "paraphilias" ],
+	[ "persona", "personae", "personæ", "personas" ],
+	[ "pharmacopoeia", "pharmacopoeiae", "pharmacopœia", "pharmacopœiæ" ],
+	[ "pinna", "pinnae", "pinnæ" ],
+	[ "planta", "plantae", "plantæ" ],
+	[ "pleura", "pleurae", "pleuræ" ],
+	[ "pupa", "pupae", "pupæ" ],
+	[ "retina", "retinae", "retinæ", "retinas" ],
+	[ "saliva", "salivae", "salivæ", "salivas" ],
+	[ "scapula", "scapulae", "scapulæ" ],
+	[ "scrofula", "scrofulae", "scrofulæ" ],
+	[ "sequela", "sequelae", "sequelæ" ],
+	[ "spatula", "spatulae", "spatulæ", "spatulas" ],
+	[ "spermatheca", "spermathecae", "spermathecæ" ],
+	[ "squama", "squamae", "squamæ" ],
+	[ "stipula", "stipulae", "stipulæ" ],
+	[ "stria", "striae", "striæ" ],
+	[ "subpoena", "subpoenae", "subpœna", "subpœnæ", "subpoenas" ],
+	[ "summa", "summae", "summæ", "summas" ],
+	[ "synaesthesia", "synaesthesiae", "synæsthesia", "synæsthesiæ", "synaesthesias" ],
+	[ "taenia", "taeniae", "taeniæ" ],
+	[ "testa", "testae", "testæ" ],
+	[ "tibia", "tibiae", "tibiæ" ],
+	[ "toga", "togae", "togæ" ],
+	[ "trachea", "tracheae", "tracheæ", "tracheas" ],
+	[ "ulna", "ulnae", "ulnæ" ],
+	[ "urethra", "urethrae", "urethræ", "urethras" ],
+	[ "uva", "uvae", "uvæ" ],
+	[ "uvula", "uvulae", "uvulæ" ],
+	[ "vagina", "vaginae", "vaginæ", "vaginas" ],
+	[ "vertebra", "vertebrae", "vertebræ", "vertebras" ],
+	[ "vesica", "vesicae", "vesicæ" ],
+	[ "via", "viae", "viæ" ],
+	[ "vibratiuncula", "vibratiunculae", "vibratiunculæ" ],
+	[ "vita", "vitae", "vitæ" ],
+	[ "vulva", "vulvae", "vulvæ", "vulvas" ],
+	[ "zona", "zonae", "zonæ" ],
+
+	// Latin: consonant+is - consonant+es
+	[ "avis", "aves" ],
+	[ "epipubis", "epipubes" ],
+	[ "hemipenis", "hemipenes" ],
+	[ "jejunoileitis", "jejunoileites" ],
+	[ "lymphadenitis", "lymphadenites", "lymphadenitides" ],
+	[ "mantis", "mantes" ],
+	[ "naris", "nares" ],
+	[ "palatonaris", "palatonares" ],
+	[ "pelvis", "pelves" ],
+	[ "penis", "penes" ],
+	[ "pileipellis", "pileipelles" ],
+	[ "postnaris", "postnares" ],
+	[ "prepubis", "prepubes" ],
+	[ "pseudopenis", "pseudopenes" ],
+	[ "pubis", "pubes" ],
+	[ "sternochondroscapularis", "sternochondroscapulares" ],
+	[ "tax", "taxi", "taxis", "taxes" ],
```

Comment:
Why is `tax` and `taxi` seen as on lemma? 🤔

Reply:
Actually, there are three lemmas there: tax - taxes, taxi - taxis, taxis - taxes. Because they overlap in forms it is convenient to see them as one lemma (and Google allows such inaccuracies itself). Technically we can accurately predict word forms for "tax" and for "taxi", but not for the other four cases.